test/extended/images/append.go

```diff
@@ -93,6 +93,9 @@ var _ = g.Describe("[sig-imageregistry][Feature:ImageAppend] Image append", func
 		pod := cli.Create(context.TODO(), cliPodWithPullSecret(oc, heredoc.Docf(`
 			set -x
 
+			# create a copy of the single manifest image from the original image for layer count
+			oc image append --insecure --from %[3]s --to %[2]s/%[1]s/tools:singlemanifest
+
```

Comment:
the ideal way would have been to tag the image - but the builder role doesn't have perms for creating istags.